src/main/java/th/co/orcsoft/training/common/db/dao/impl/CenterPointDaoImpl.java

```diff
@@ -13,7 +13,7 @@
 
 	@Override
 	public List<VoteModel> getRequestedConfirmations() {
-		String sql = "SELECT * FROM VOTE WHERE AprvBy = null";
+		String sql = "SELECT * FROM VOTE WHERE AprvBy = null AND UpdBy = true AND UpdAprvBy = null";
```

Comment:
SELECT * FROM VOTE WHERE AprvFlag is null and UpdAprvFlag is null and UpdFlag = 'true';